### DISCUSSION TITLE
Running on M2 Mac mini + 26GB RAM. Very Very slow

### DISCUSSION THREAD

**Discussion by RobertChalmers** [openai/whisper#1518] [General]
So apparently the M2 Silicone Mac doesn't have a recognizable GPU, and DEVICE defaults to CPU. Which of course means its running incredibly slowly.
This is a fast computer otherwise, and I'm wondering if there are plans to update the build to take advantage of the M2 Silicone processing power?

**Comment by phineas-pta** [comment]
see #984
